cmake/Modules/Findursa.cmake

```diff
@@ -0,0 +1,26 @@
+add_library(ursa UNKNOWN IMPORTED)
+
+set(URL "https://github.com/hyperledger/ursa/")
+set(VERSION "80bc3e772f7217c0a6c51507c0ddcd9c8bc77c0d")
+
+set(URSA_BUILD ${CMAKE_CURRENT_BINARY_DIR}/libursa)
+
+set(URSA_SRC ${EP_PREFIX}/src/hyperledger_ursa)
+set(URSA_LIB ${URSA_BUILD}/lib)
+
+externalproject_add(hyperledger_ursa
+  GIT_REPOSITORY ${URL}
+  GIT_TAG        ${VERSION}
+  CONFIGURE_COMMAND "" # remove configure step
+  BUILD_IN_SOURCE 1
+  BUILD_COMMAND cargo build --release --no-default-features --features="ffi"
+  INSTALL_COMMAND ""
+  UPDATE_COMMAND ""
+  INSTALL_COMMAND
+    mkdir -p ${URSA_LIB} &&
+    cp -r ${URSA_SRC}/target/release/libursa.so ${URSA_LIB} &&
+    cp -r ${URSA_SRC}/libursa/include ${URSA_BUILD}
+)
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/libursa/include)
```

Comment:
It is better not to use global `include_` and `link_` calls so that we can explicitly define which targets should include and link Ursa code. Since you have defined `ursa` library target in the first line, you should be able to use `set_target_properties` for `ursa` like it is done in `gflags` for example: https://github.com/hyperledger/iroha/blob/8917810fd5ef8657a98da9db051f973c9ef73c39/cmake/Modules/Findgflags.cmake#L38-L41